components/kyma-environment-broker/cmd/broker/broker_suite_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	gardenerFake "github.com/gardener/gardener/pkg/client/core/clientset/versioned/fake"
 	"github.com/google/uuid"
 	"github.com/gorilla/mux"
+	contract "github.com/kyma-incubator/reconciler/pkg/keb"
```

Comment:
i'm not entirely sure about this naming convention - let's discuss it f2f

Reply:
I don't like "contract", for the the naming is very, very strange. Why reconciler people named the package "keb". It should be "api" because it is an api of reconciler. In our application it must not be "contrat". If I ask you what does "contract" mean nobody knows it is about reconciler.